homeassistant/components/climate/strings.json

```diff
@@ -298,6 +298,20 @@
   },
   "title": "Climate",
   "triggers": {
+    "hvac_mode_changed": {
+      "description": "Triggers after the mode of one or more climate-control devices change.",
+      "fields": {
+        "behavior": {
+          "description": "[%key:component::climate::common::trigger_behavior_description%]",
+          "name": "[%key:component::climate::common::trigger_behavior_name%]"
+        },
+        "hvac_mode": {
+          "description": "The HVAC modes to trigger on.",
+          "name": "Modes"
+        }
+      },
+      "name": "Climate-control device mode changed"
+    },
```

Comment:
I'm not sure about the language here. We will (although not yet enabled the UI) allow multiple to-states.
We usually use singular when both single and multiple are allowed (`hvac_mode` not `hvac_modes`), but I think natural language should be plural?

Reply:
The device will only have one mode at a time, which was what changed, so I think it should be singular.

Reply:
Right, the name of the trigger (and the key, `climate.hvac_mode_changed`) should be singular.
But what about the configuration key `hvac_mode`?

Reply:
OK, then it's fine as is